main.go

```diff
@@ -268,6 +268,27 @@ func run() int {
 	app.Usage = "Memo Life For You"
 	app.Version = VERSION
 	app.Commands = commands
+	app.Action = func(c *cli.Context) error {
+		args := c.Args()
+		if args.Present() {
+			xcmd := fmt.Sprintf("memo-%s", args.First())
+			xcmdpath, err := exec.LookPath(xcmd)
```

Comment:
I hope to make limit that such plugins have to be put on ~/.config/memo/plugins.